src/lib/errors.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-export function L10nError(message, id, code) {
-  this.name = 'L10nError';
-  this.message = message;
-  this.id = id;
-  this.code = code;
+export class L10nError extends Error {
+  constructor(message, id, code) {
+    this.name = 'L10nError';
```

Comment:
Did you try to wepack this? :)  You'll need to call `super` before you can use `this` in the constructor.